README.md

```diff
@@ -352,19 +368,28 @@ const encoded = = encode({myType: new MyType<any>()}, { extensionCodec, context
 const decoded = decode(encoded, { extensionCodec, context });
 ```
 
-#### Handling BigInt with ExtensionCodec
+#### Handling BigInt and big numbers
+
+**Decoding**
 
-This library does not handle BigInt by default, but you have two options to handle it:
+This library does not handle decoding BigInt by default, but you have three options to handle it:
```

Comment:
Actually we default to `IntMode.AS_ENCODED` if it's not specified, so this is no longer accurate

Reply:
Default is actually `UNSAFE_NUMBER`

Reply:
Good point, I read it wrong